src/schema/v2/partnerInquirerProfile.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  GraphQLBoolean,
+  GraphQLFieldConfig,
+  GraphQLFieldConfigMap,
+  GraphQLID,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import date from "./fields/date"
+import Image, { normalizeImageData } from "./image"
+import { CollectorProfileFields } from "./me/collector_profile"
+import { myLocationType } from "./me/myLocation"
+
+export const CollectorProfileArtists = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "CollectorProfileArtists",
+  fields: {
+    name: { type: GraphQLString },
+  },
+})
+
+const InquirerCollectorProfileFields: GraphQLFieldConfigMap<
+  any,
+  ResolverContext
+> = {
+  ...CollectorProfileFields,
+  location: { type: myLocationType },
+  artsyUserSince: date,
+  ownerID: {
+    type: new GraphQLNonNull(GraphQLID),
+    description: "User ID of the collector profile's owner",
+    resolve: ({ owner: { id } }) => id,
+  },
+  icon: {
+    type: Image.type,
+    resolve: ({ icon }) => normalizeImageData(icon),
+  },
+  bio: {
+    type: GraphQLString,
+  },
+  profession: { type: GraphQLString },
+  otherRelevantPositions: {
+    type: GraphQLString,
+    description: "Collector's position with relevant institutions",
+    resolve: ({ other_relevant_positions }) => other_relevant_positions,
+  },
+  emailVerified: {
```

Comment:
Let's maybe rename this to `emailConfirmed` to match the me route. See https://github.com/artsy/metaphysics/blob/ad9fc1d6aaffc1ee4c625c616e4fe46db2a16e23/_schemaV2.graphql#L8797